source/_layouts/base.liquid

```diff
@@ -71,21 +78,23 @@
         </h1>
       {% endunless %}
 
-      <div class="sl-l-container sl-color--white-background{% unless is_home %} sl-l-section{% endunless %}">
+      <div class="sl-l-container sl-color--white-background{% unless is_home %} sl-l-section{% endunless %}{% if is_playground %} sl-l-section--playground{% endif %}">
         {{ content | typogr }}
       </div>
 
-      <div class="sl-c-alert">
-        <div class="sl-l-container sl-c-list-horizontal-wrapper" aria-labelledby="release-nav">
-          <ul class="sl-l-grid--justify-center">
-            <li id="release-nav">Current Releases:</li>
-            <li><span class="release-name"><a href="/dart-sass">Dart Sass</a> <a href="{{ releases['dart-sass'].url }}">{{ releases['dart-sass'].version }}</a></span></li>
-            <li><span class="release-name"><a href="/libsass">LibSass</a> <a href="{{ releases['libsass'].url }}">{{ releases['libsass'].version }}</a></span></li>
-            <li><span class="release-name"><a href="/ruby-sass">Ruby Sass</a> <span aria-label="coffin" role="presentation">⚰</span></span></li>
-            <li class="sl-l-grid__column sl-l-large-grid__column--auto-size"><a href="/implementation">Implementation Guide</a></li>
-          </ul>
+      {% unless is_playground %}
```

Comment:
It might make sense to add new layouts that extend base (like a playground-specific layout), rather than putting all of it in the base layout?